.gitignore

```diff
@@ -100,5 +100,9 @@ venv.bak/
 # mkdocs documentation
 /site
 
+# Editors
+.idea/
```

Comment:
Drive-by: This belongs in your [global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

Reply:
Done!